aspnetcore/tutorials/first-web-api/sample/src/TodoApi/Startup.cs

```diff
@@ -26,7 +26,11 @@ public Startup(IHostingEnvironment env)
         public void ConfigureServices(IServiceCollection services)
         {
             // Add framework services.
-            services.AddMvc();
+            services.AddMvcCore()
+                .AddAuthorization()
```

Comment:
For our intro to building Web APIs tutorial I think we should stick with `AddMvc` as it creates the least amount of friction.